modules/backend/clinic/staff/receptionist/ReceptionistMock.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "iReceptionist.hpp"
+#include <gmock/gmock.h>
+
+namespace clinic
+{
+namespace staff
+{
+namespace receptionist
+{
+
+class ReceptionistMock : public iReceptionist
```

Comment:
I think, that Mocks should reside in "tests" module, not in backend. It does not need to be included in production code in my opinion, as it is part of "testing" world.

The best option in my opinion is to create folder "Mocks" or "utils/mocks"  in "tests" module

Reply:
I thought about it also but wasn't sure. In different projects I followed this approach, so I added it here also. Will move it to utils/mocks under the test module